src/common/options.cc

```diff
@@ -7047,6 +7047,10 @@ std::vector<Option> get_mds_client_options() {
     .set_default(false)
     .set_description("big_writes is deprecated in libfuse 3.0.0"),
 
+    Option("fuse_max_write", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    .set_default(0)
+    .set_description(""),
```

Comment:
Please also add a description. You can use what you wrote in the docs.

Reply:
updated it.